qutebrowser/keyinput/modeparsers.py

```diff
@@ -215,7 +215,6 @@ def handle(self, e: QKeyEvent, *,
         if (self._command_parser.handle(e, dry_run=True) !=
                 QKeySequence.NoMatch):
             log.keyboard.debug("Handling key via command parser")
-            self.clear_keystring()
```

Comment:
huh?

Reply:
So you can cycle through visible hints when you've typed a partial hint string